docker/README.md

```diff
@@ -0,0 +1,168 @@
+- `TAG=<tag>` is used to determine the tag of the container image, *e.g.* `stable`. In the argument is not specified, it will default to the value set by `KUBESTELLAR_VERSION`.
+- `PLATFORMS` lists the desired target platforms. Default value; `PLATFORMS=linux/amd64,linux/arm64,linux/ppc64le`. It should be noted that a `linux/s390x` **KubeStellar**  cannot be build because **kcp** does not have a `linux/s390x` binary release.
+
+As an example, the **KubeStellar** container images available at https://quay.io/repository/kubestellar/kubestellar?tab=tags have been built with commands like:
+
+```bash
+make buildx IMG=quay.io/kubestellar/kubestellar KUBESTELLAR_VERSION=v0.4.0
+```
+
+## Run the container and access KubeStellar from the host OS
+
+Spin up a local or remote container image with the command:
+
+```bash
+make run
+export KUBECONFIG=${HOME}/.kcp/admin.kubeconfig
+export PATH=$PATH:${HOME}/kcp:${HOME}/kubestellar/bin
+```
+
+The following arguments can be used:
+
+- `IMG=[<registry>/<organization>/]<image>` select the container image, the default value is the local `kubestellar` image. Use `IMG=quay.io/kubestellar/kubestellar` to pull the **stable** container image from our [quay.io registry](https://quay.io/repository/kubestellar/kubestellar?tab=tags).
+- `TAG=<tag>`, select the image tag, the default value is the **stable** version of **KubeStellar** as specified in https://github.com/kubestellar/kubestellar/blob/main/VERSION.
+- `NAME=<container-name>` sets the name of the container, the default value is `kubestellar`
+- `BASEPATH=<path>` set the base path where the **KUBECONFIG** and executable will be made available. By default, `~` is used. Note that this parameter will impact the exact path of the `export` commands above, which are provided for the default case. The `make run` command will print out the correct export commands to be used.
+
+The command above will:
+
+- share a `${BASEPATH}/.kcp` folder containing the `KUBECONFIG` file
+- share a `${BASEPATH}/.kubestellar-logs` folder containing the log files of **kcp** and **KubeStellar** controllers
+- create a `${BASEPATH}/kcp` folder containing a copy of **kcp** plugins for the host OS architecture
+- create a `${BASEPATH}/kubestellar` folder containing a copy of **KubeStellar** executables for the host OS architecture
+
+At this point, we can check basic **KubeStellar** functionality with the command:
+
+```bash
+kubectl ws root
+kubectl ws tree
+```
+
+which should return something like:
+
+```text
+.
+└── root
+    ├── compute
+    └── espw
+```
+
+At this point, one can follow the instructions and examples in **KubeStellar** [Quickstart](https://docs.kubestellar.io/release-0.3/Getting-Started/quickstart/).
+
+## Checking the container logs
+
+The running container logs can be easily displayed using the command:

```

Comment:
It is true that the following command displays _some_ log from the container, but it is none of the most interesting ones.  The kcp server log and the KubeStellar controller logs are _not_ included.  But the wording here does not make that clear.  Perhaps a remark about the logs directory mentioned above is in order here.

Reply:
@francostellari I think I agree with @MikeSpreitzer here.  Let's be specific that the logs being served are from the container and not from kcp or kubestellar.  I think that is a good idea.

Reply:
@clubanderson perhaps it's my shortcoming with English but the quoted text above from README.md seems to say exactly what you are saying. To avoid issues I have removed the section from the makefile and README.md